whylog/constraints/verifier.py

```diff
@@ -8,15 +8,18 @@
 
 class ConstraintRegistry(object):
     CONSTRAINTS = {
-        'identical': IdenticalConstraint(params_checking=False),
-        'time': TimeConstraint(params_checking=False)
+        'identical': IdenticalConstraint,
+        'time': TimeConstraint
         # register your constraint here
     }
 
     @classmethod
     def get_constraint(cls, constraint_data):
         if constraint_data['name'] in cls.CONSTRAINTS:
-            return cls.CONSTRAINTS[constraint_data['name']]
+            return cls.CONSTRAINTS[constraint_data['name']](
+                param_dict=constraint_data['params'],
+                params_checking=False
```

Comment:
`params_checking` is option which should be enabled only in teacher module. constraints verification should not use this option.